src/fsharp/fsharp.core.netcore.nuget/layoutfscorenuget.fsx

```diff
@@ -57,7 +57,7 @@ try
         if not (Directory.Exists(output)) then Directory.CreateDirectory(output) |>ignore
         ()
 
-    let fsharpCompilerFiles =
+    let fsharpCoreFiles =
         seq {
             yield Path.Combine(bindir, "FSharp.Core.dll")
             yield Path.Combine(bindir, "fsharp.core.sigdata")
```

Comment:
Why is ``fsharp.core.sigdata`` lower case here?  On Linux we'd expect this file to be called ``FSharp.Core.sigdata``?